mix.exs

```diff
@@ -31,7 +31,7 @@ defmodule TelemetryMetricsPrometheus.Core.MixProject do
 
   defp deps do
     [
-      {:telemetry_metrics, "~> 0.6"},
+      {:telemetry_metrics, "~> 0.6 or ~> 1.0.0"},
```

Comment:
```suggestion
      {:telemetry_metrics, "~> 0.6 or ~> 1.0"},
```

Reply:
Made this update. My apologies, I admittedly don't fully understand the nuances of the Elixir versioning system.
If you don't mind my asking, why is it that we are preferring 1.0 here instead of 1.0.0?

Reading through [these docs](https://hexdocs.pm/elixir/Version.html#module-requirements) I was under the impression that `~> 1.0.0` would have the desired outcome (>= 1.0.0 and < 1.1.0), whereas `~> 1.0` would allow for any version until the next major version (>= 1.0.0 and < 2.0.0).

Am I missing something? Always happy to learn more 🙂

Reply:
There are no breaking changes after 1.0 so we don't lock it to a minor version. That would then require publishing new packages every time a minor change is made to that library and can create dependency version conflicts for users.